lib/event.js

```diff
@@ -346,6 +346,21 @@ var ICalEvent = function(_data, calendar) {
             });
         }
 
+        if(repeating.exclude) {
+            if(!Array.isArray(repeating.exclude)) {
+                repeating.exclude = [repeating.exclude];
+            }
+
+            data.repeating.exclude = [];
+            repeating.exclude.forEach(function(excludedDate) {
+                if(Object.prototype.toString.call(excludedDate) === '[object Date]') {
```

Comment:
I feel like there should be a better way to check for date objects here. I would do `instanceof` but that doesn't necessarily always work if you are crossing frames, as the global `Date` objects will be different (http://stackoverflow.com/a/643827).